eclass/mate-desktop.org.eclass

```diff
@@ -50,15 +50,11 @@ inherit versionator
 
 # Set SRC_URI or EGIT_REPO_URI based on whether live
 if [[ ${PV} == 9999 ]]; then
-	EGIT_REPO_URI="
-		https://github.com/mate-desktop/${MATE_DESKTOP_ORG_PN}.git
-		git://github.com/mate-desktop/${MATE_DESKTOP_ORG_PN}.git
-		http://github.com/mate-desktop/${MATE_DESKTOP_ORG_PN}.git
-	"
+	EGIT_REPO_URI="https://github.com/mate-desktop/${MATE_DESKTOP_ORG_PN}.git"
```

Comment:
@gentoo/mate, do you agree with this? I suppose you might have had the fallbacks for a reason, though plain `http://` seems like a horrible choice for GitHub (if it even works).